docs/developer/set-goals.md

```diff
@@ -79,11 +79,47 @@ The DSL includes support for logical operations. For example, this will build al
 whenPushSatisfies(IsMaven, not(IsKotlin)).setGoals(BuildGoals),
 ```
 
-## PushRule
-
 Each argument to [`sdm.withPushRules`](https://atomist.github.io/sdm/interfaces/_lib_api_machine_softwaredeliverymachine_.softwaredeliverymachine.html#withpushrules)
 is a PushRule, contributing goals on a commit if a condition is met. That condition is a PushTest.
 
+### Advanced Push Rules
+
+#### Computed Values
+
+You can use computed `boolean` values or the results of synchronous or
+asynchronous functions returning `boolean` in the DSL, making it
+possible to bring in any state you wish. For example:
+
+```typescript
+whenPushSatisfies(IsMaven, HasSpringBootApplicationClass,
+	deploymentsToday < 25)
```

Comment:
that's a constant boolean.
Do you mean `async (inv) => deploymentsToday < 25` ?